x-pack/plugins/observability_solution/investigate_app/server/models/investigation.ts

```diff
@@ -8,5 +8,4 @@
 import { investigationSchema } from '@kbn/investigation-shared';
 import * as t from 'io-ts';
 
-export type Investigation = t.TypeOf<typeof investigationSchema>;
```

Comment:
Did not mean to delete this at all. Don't know what happened here. Probably tried to copy and accidentally deleted.